lib/ExportsInfo.js

```diff
@@ -137,18 +137,21 @@ class ExportsInfo {
 
 	_sortExportsMap(exports) {
 		if (exports.size > 1) {
-			const entriesInOrder = Array.from(exports.values());
-			if (
-				entriesInOrder.length !== 2 ||
-				entriesInOrder[0].name > entriesInOrder[1].name
-			) {
-				entriesInOrder.sort((a, b) => {
-					return a.name < b.name ? -1 : 1;
-				});
-				exports.clear();
-				for (const entry of entriesInOrder) {
-					exports.set(entry.name, entry);
-				}
+			const namesInOrder = [];
+			for (const entry of exports.values()) {
+				namesInOrder.push(entry.name);
+			}
+			namesInOrder.sort();
+			let i = 0;
+			for (const entry of exports.values()) {
+				const name = namesInOrder[i];
+				if (entry.name !== name) break;
+			}
```

Comment:
This is missing an increment to `i` somewhere

Reply:
Oh yes. Do you want to send a PR